### PR TITLE
Add Query: Find mutation count across chromosome

### PR DESCRIPTION
**Pull request by tsa87**
The query finds the numbers of mutation positions across a specific region of chromosome with a bin size of 100,000 base pairs. It currently loops through each interval and collect the mutation count.

### CODE REVIEW

**Comment by tsa87** [comment]
@googlebot I signed it!

File: docs/sample_queries/gnomad/gnomad.ipynb [hunk 1]
```diff
@@ -6824,6 +6824,150 @@
...
+            "fd6f1303ad6a454cabac426f14c78f16",
+            "4203f98b3a4b4723b398310a4691cb8c",
+            "a54bb74bb77244b2a74ad456b0a5f12f",
+            "03805018509243698bc8ac65990ce734"
+          ]
+        }
+      },
+      "source": [
+        "query_template = \"\"\"\n",
+        "SELECT count(1) AS num_mutations\n",
```
Comment (samanvp) [inline docs/sample_queries/gnomad/gnomad.ipynb:6932]: Please update the query cell as follows:
```
bucket_size = 10000
query_template = """
SELECT CAST(FLOOR(DIV(start_position, {BUCKET})) AS INT64) AS start_pos_bucket ,
       count(1) AS num_snv
FROM 
    (
    SELECT DISTINCT 
          start_position,
          alternate_bases.alt AS alt,
          {VAR_TYPE_COL} AS variant_type,
    FROM `bigquery-public-data.gnomAD.{GNOMAD_VER}__{CHROM}` AS main_table,
        main_table.alternate_bases AS alternate_bases
    WHERE variant_type = 'snv'
    )
GROUP BY 1
ORDER BY 1
"""
query = query_template.format(GNOMAD_VER=gnomad_version_chr,
                              CHROM=chromosome_chr,
                              VAR_TYPE_COL=variant_type_col,
                              BUCKET=bucket_size)

snv_dist = run_query(query)
snv_dist.head()

```

Comment (samanvp) [reply docs/sample_queries/gnomad/gnomad.ipynb:6932]: Then add a second cell to visualize the result:
```
import matplotlib.pyplot as plt

plt.figure(dpi=150)
plt.bar(snv_dist.start_pos_bucket, snv_dist.num_snv)
plt.xlabel("Bucket number of start_pos")
plt.ylabel("No of SNVs in each bucket")
plt.title("Distribution of SNVs on {} for buckets of {} base pairs".format(chromosome_chr, bucket_size))
plt.show()
```

**Review by samanvp — APPROVED** [review]
Please make sure you update your branch. We just pushed a few modifications to the notebook.

File: docs/sample_queries/gnomad/gnomad.ipynb (hunk 1 above)
Comment (tsa87) [reply docs/sample_queries/gnomad/gnomad.ipynb]: Hi Saman, thank you so much for these suggestions. I have updated my PR, have a good day!

**Comment by samanvp** [comment]
Thank you very much Tony for contributing to this project.

The idea of this notebook started from [the project](https://github.com/tsa87/gnomAD-mutation-query) you initiated during your internship at Google.

**Pull request merged**